.esbuild/build.ts

```diff
@@ -52,9 +53,12 @@ const handler = (e) => {
 };
 
 const main = async () => {
+  generateLangium();
   await mkdir('stats').catch(() => {});
   const packageNames = Object.keys(packageOptions) as (keyof typeof packageOptions)[];
-  await Promise.allSettled(packageNames.map((pkg) => buildPackage(pkg).catch(handler)));
+  for (const pkg of packageNames) {
+    await buildPackage(pkg).catch(handler);
+  }
```

Comment:
> Is this change on purpose?

Yes, with the old approach it tries to build `mermaid` before `parser` finishes which throws error:

![4ECF5452-035E-464F-8B67-154DD6D08AF6](https://github.com/mermaid-js/mermaid/assets/87622592/08e64312-1979-49de-a48a-c5aca3672378)

> If it is on purpose, we might want to add a comment on why we're building these packages sequentially, instead of in parallel.

Will do 👍.

Reply:
Added a comment in 8725c829a